### ISSUE TITLE
semantic filter for the hashtag neighbourhood

### ISSUE THREAD

**Issue by djelouze** [Meewan/MercrediFiction#14]
What about filtering toots that use the #mercredifiction in a sentence ? It generally means that the tooter is actually talking about mercredifiction, not telling a story.

ex: 
https://witches.town/@Ewakhine/1891335
https://mamot.fr/@guytarr/1774514
https://mamot.fr/@TigtaK/1758820
...

**Comment by Meewan** [comment]
Good idea but I have no idea on how to do that. The only easy way would be if it is between a capital and a dot but I am not sure it is enough.

(It must be a light method like a simple regex. I cannot do hard computing on each toot)
